2019-08-05 12:11:25.77 [ERR] [session.go:246] recvLoop error: read tcp 127.0.0.1:17000->127.0.0.1:54039: wsarecv: An existing connection was forcibly closed by the remote host., remoteAddr: 127.0.0.1:54039, sessionid: 1
2019-08-05 12:11:25.77 [ERR] [session.go:254] read and handle message error: read tcp 127.0.0.1:17000->127.0.0.1:54039: wsarecv: An existing connection was forcibly closed by the remote host., remoteAddr: 127.0.0.1:54039, sessionid: 1
2019-08-05 12:11:25.77 [INF] [serviceProc.go:105] Proc_ServiceA OnClosed: 1 127.0.0.1:54039
2019-08-05 12:11:25.771 [DBG] [GateUserBase.go:165] lobby user OnClosed, uid: 1
2019-08-05 12:12:31.716 [INF] [app.go:139] services[serviceA]
2019-08-05 12:12:31.723 [DBG] [app.go:76] App.init, appID: 35
2019-08-05 12:12:31.764 [INF] [serviceA.go:24] ServiceA OnInit
2019-08-05 12:12:31.833 [DBG] [serviceProc.go:149] set verify msgid, err: 0
2019-08-05 12:12:31.833 [DBG] [serviceProc.go:153] outer addr: 127.0.0.1:17000
2019-08-05 12:12:31.833 [DBG] [localService.go:70] service check : serviceA
2019-08-05 12:12:31.833 [DBG] [localService.go:100] service check success: serviceA
2019-08-05 12:12:31.833 [DBG] [localServiceMgr.go:50] App InitLocalService, name: serviceA
2019-08-05 12:12:31.917 [DBG] [localService.go:185] run service , serviceName: serviceA, serviceType: 2, ServerID: 36 tickMS: 2000
2019-08-05 12:12:35.346 [DBG] [serviceProc.go:67] MsgProc_LoginReq, uid: 1
2019-08-05 12:12:44.878 [DBG] [dbservice.go:52] connect failed,no reachable servers
2019-08-05 12:12:44.878 [ERR] [mongodboperate.go:89] session nil, not connect or auth
2019-08-05 12:12:44.878 [INF] [Entity_Props.go:171] select props: map[coin:1 exp:1 level:1 rating:1]props return: map[]
2019-08-05 12:12:44.878 [DBG] [entitySrvUtil.go:64] Reg service key: entitysrvinfo:1, serviceType: 2, serviceID: 36
2019-08-05 12:12:44.88 [DBG] [entitySrvUtil.go:64] Reg service key: entitysrvinfo:1, serviceType: 1, serviceID: 36
2019-08-05 12:12:44.881 [DBG] [ServiceAUser.go:20] ServiceAUser.OnUserInit, dbid: 1 isnew: true
2019-08-05 12:12:44.881 [INF] [serviceA.go:52] ServiceA OnLoginHandler, msg.UID = 1
2019-08-05 12:12:44.882 [DBG] [ServiceAUserRpcProc.go:14] RPC_Hello, name: hello world, id: 1
2019-08-05 12:12:44.882 [DBG] [ServiceAUserRpcProc.go:16] err: = <nil>
2019-08-05 12:17:39.662 [INF] [app.go:139] services[serviceA]
2019-08-05 12:17:39.669 [DBG] [app.go:76] App.init, appID: 37
2019-08-05 12:17:39.711 [INF] [serviceA.go:24] ServiceA OnInit
2019-08-05 12:17:39.783 [DBG] [serviceProc.go:149] set verify msgid, err: 0
2019-08-05 12:17:39.783 [DBG] [serviceProc.go:153] outer addr: 127.0.0.1:17000
2019-08-05 12:17:39.783 [DBG] [localService.go:70] service check : serviceA
2019-08-05 12:17:39.783 [DBG] [localService.go:100] service check success: serviceA
2019-08-05 12:17:39.783 [DBG] [localServiceMgr.go:50] App InitLocalService, name: serviceA
2019-08-05 12:17:39.866 [DBG] [localService.go:185] run service , serviceName: serviceA, serviceType: 2, ServerID: 38 tickMS: 2000
2019-08-05 12:17:43.295 [DBG] [serviceProc.go:67] MsgProc_LoginReq, uid: 1
2019-08-05 12:17:52.819 [DBG] [dbservice.go:52] connect failed,no reachable servers
2019-08-05 12:17:52.819 [ERR] [mongodboperate.go:89] session nil, not connect or auth
2019-08-05 12:17:52.819 [INF] [Entity_Props.go:171] select props: map[coin:1 exp:1 level:1 rating:1]props return: map[]
2019-08-05 12:17:52.819 [DBG] [entitySrvUtil.go:64] Reg service key: entitysrvinfo:1, serviceType: 2, serviceID: 38
2019-08-05 12:17:52.82 [DBG] [entitySrvUtil.go:64] Reg service key: entitysrvinfo:1, serviceType: 1, serviceID: 38
2019-08-05 12:17:52.822 [DBG] [ServiceAUser.go:20] ServiceAUser.OnUserInit, dbid: 1 isnew: true
2019-08-05 12:17:52.822 [INF] [serviceA.go:52] ServiceA OnLoginHandler, msg.UID = 1
2019-08-05 12:17:52.822 [DBG] [ServiceAUserRpcProc.go:14] RPC_Hello, name: hello world, id: 1
2019-08-05 12:17:52.822 [DBG] [ServiceAUserRpcProc.go:16] err: = <nil>
2019-08-05 12:18:34.938 [ERR] [session.go:246] recvLoop error: read tcp 127.0.0.1:17000->127.0.0.1:54444: wsarecv: An existing connection was forcibly closed by the remote host., remoteAddr: 127.0.0.1:54444, sessionid: 1
2019-08-05 12:18:34.939 [ERR] [session.go:254] read and handle message error: read tcp 127.0.0.1:17000->127.0.0.1:54444: wsarecv: An existing connection was forcibly closed by the remote host., remoteAddr: 127.0.0.1:54444, sessionid: 1
2019-08-05 12:18:34.939 [INF] [serviceProc.go:105] Proc_ServiceA OnClosed: 1 127.0.0.1:54444
2019-08-05 12:18:34.94 [DBG] [GateUserBase.go:165] lobby user OnClosed, uid: 1
2019-08-05 12:18:52.241 [INF] [app.go:139] services[serviceA]
2019-08-05 12:18:52.308 [DBG] [app.go:76] App.init, appID: 39
2019-08-05 12:18:52.309 [INF] [serviceA.go:24] ServiceA OnInit
2019-08-05 12:18:52.374 [DBG] [serviceProc.go:149] set verify msgid, err: 0
2019-08-05 12:18:52.374 [DBG] [serviceProc.go:153] outer addr: 127.0.0.1:17000
2019-08-05 12:18:52.374 [DBG] [localService.go:70] service check : serviceA
2019-08-05 12:18:52.374 [DBG] [localService.go:100] service check success: serviceA
2019-08-05 12:18:52.374 [DBG] [localServiceMgr.go:50] App InitLocalService, name: serviceA
2019-08-05 12:18:52.455 [DBG] [localService.go:185] run service , serviceName: serviceA, serviceType: 2, ServerID: 40 tickMS: 2000
2019-08-05 12:18:54.988 [DBG] [serviceProc.go:67] MsgProc_LoginReq, uid: 1
2019-08-05 12:19:04.523 [DBG] [dbservice.go:52] connect failed,no reachable servers
2019-08-05 12:19:04.523 [ERR] [mongodboperate.go:89] session nil, not connect or auth
2019-08-05 12:19:04.523 [INF] [Entity_Props.go:171] select props: map[coin:1 exp:1 level:1 rating:1]props return: map[]
2019-08-05 12:19:04.523 [DBG] [entitySrvUtil.go:64] Reg service key: entitysrvinfo:1, serviceType: 2, serviceID: 40
2019-08-05 12:19:04.524 [DBG] [entitySrvUtil.go:64] Reg service key: entitysrvinfo:1, serviceType: 1, serviceID: 40
2019-08-05 12:19:04.526 [DBG] [ServiceAUser.go:20] ServiceAUser.OnUserInit, dbid: 1 isnew: true
2019-08-05 12:19:04.526 [INF] [serviceA.go:52] ServiceA OnLoginHandler, msg.UID = 1
2019-08-05 12:19:04.526 [DBG] [ServiceAUserRpcProc.go:14] RPC_Hello, name: hello world, id: 1
2019-08-05 12:19:04.526 [DBG] [ServiceAUserRpcProc.go:16] err: = <nil>
